tests/test_text_generation_example.py

```diff
@@ -61,6 +61,8 @@
             ("baichuan-inc/Baichuan2-7B-Chat", 1, True, 108, False),
             ("baichuan-inc/Baichuan2-13B-Chat", 1, False, 66, False),
             ("deepseek-ai/DeepSeek-V2-Lite", 1, False, 35, False),
+            ("THUDM/chatglm2-6b", 1, True, 150, False),
+            ("THUDM/chatglm3-6b", 1, True, 150, False),
```

Comment:
What's the difference between ChatGLM-2 and ChatGLM-3 exactly? To know if we really need to test both

Reply:
I don't think there is modeling functional difference, the only difference lies in some customized tokenizer methods' implementation. I removed the test for chatglm2.